From f1fc09295014a5210fedffeb5b1ce4e8512276ce Mon Sep 17 00:00:00 2001
From: olegsvs <oleg.texet@gmail.com>
Date: Tue, 10 May 2016 21:15:40 +0300
Subject: [PATCH 035/117] JD9365 ver1

---
 include/generated/compile.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/generated/compile.h b/include/generated/compile.h
index 53f184c..9eb54c0 100644
--- a/include/generated/compile.h
+++ b/include/generated/compile.h
@@ -1,7 +1,7 @@
-/* This file is auto generated, version 13 */
+/* This file is auto generated, version 14 */
 /* SMP PREEMPT */
 #define UTS_MACHINE "arm64"
-#define UTS_VERSION "#13 SMP PREEMPT Mon May 9 22:16:34 MSK 2016"
+#define UTS_VERSION "#14 SMP PREEMPT Mon May 9 22:18:04 MSK 2016"
 #define LINUX_COMPILE_BY "olegsvs"
 #define LINUX_COMPILE_HOST "olegsvs-develop"
 #define LINUX_COMPILER "gcc version 4.9 20150123 (prerelease) (GCC) "
-- 
2.7.4

