From e90ce64d992a8ea3ef30cb943e6f77984d02f7b7 Mon Sep 17 00:00:00 2001
From: olegsvs <oleg.texet@gmail.com>
Date: Fri, 13 May 2016 21:56:12 +0300
Subject: [PATCH 073/117] clean git

---
 .gitignore | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/.gitignore b/.gitignore
index 438c52c..faa1855 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1574,6 +1574,15 @@ sound/usb/misc/modules.order
 sound/usb/modules.order
 sound/usb/usx2y/modules.order
 usr/modules.order
+include/config/auto.conf
+include/config/kernel.release
+include/generated/autoconf.h
+include/generated/compile.h
+include/generated/utsrelease.h
+
+
+
+
 
 
 
-- 
2.7.4

