From 7b42b1c1e5d0c99115ee30c769972ff8a1ad09b6 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Thu, 29 Jan 2015 17:42:36 -0800
Subject: [PATCH 0083/1168] Linux 3.10.67

Signed-off-by: Stefan Guendhoer <stefan@guendhoer.com>

Former-commit-id: 4c147278c637864b2e207761387e74427bf9e526
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 4dabaca..a88b1f9 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 VERSION = 3
 PATCHLEVEL = 10
-SUBLEVEL = 66
+SUBLEVEL = 67
 EXTRAVERSION =
 NAME = TOSSUG Baby Fish
 
-- 
2.7.4

