From fc3c023c1efea263c392eba169ada963061659d4 Mon Sep 17 00:00:00 2001
From: SnowCatPDA <snowcat@vpro.ru>
Date: Sat, 28 May 2016 15:37:43 +0500
Subject: [PATCH 110/117] UPDATE .gitignore

---
 .gitignore | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.gitignore b/.gitignore
index 3514a78..b129b7a 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,5 +1,5 @@
 *.~
-*.*~
+*~
 *.old
 *.bak
 *.orig
-- 
2.7.4

