From e5abe11579e7e60bf09259b8673f08e5b3a693a6 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Mon, 22 Jun 2015 16:56:08 -0700
Subject: [PATCH 0590/1168] Linux 3.10.81

Signed-off-by: Stefan Guendhoer <stefan@guendhoer.com>

Former-commit-id: d9234b68c760b5d7c5c69e1722731e55c02aac81
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index e2d8071..4c7f9be 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 VERSION = 3
 PATCHLEVEL = 10
-SUBLEVEL = 80
+SUBLEVEL = 81
 EXTRAVERSION =
 NAME = TOSSUG Baby Fish
 
-- 
2.7.4

