From 185e1e3932bf08fde379b3bc8c9f533ba6229bf1 Mon Sep 17 00:00:00 2001
From: olegsvs <oleg.texet@gmail.com>
Date: Tue, 10 May 2016 21:49:01 +0300
Subject: [PATCH 1092/1168] fix typo

---
 include/linux/pocket_mod.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/pocket_mod.h b/include/linux/pocket_mod.h
index 9756f40..248e639 100644
--- a/include/linux/pocket_mod.h
+++ b/include/linux/pocket_mod.h
@@ -6,7 +6,7 @@ extern int is_screen_on;
 #ifdef CONFIG_MTK_STK3X1X_DRIVER
 int stk3x1x_pocket_detection_check(void);
 #endif
-#ifdef CONFIG_MTK_epl2182_NEW
+#ifdef CONFIG_MTK_EPL2182_NEW
 int epl2182_pocket_detection_check(void);
 #endif
 int device_is_pocketed(void);
-- 
2.7.4

