From 4ec1d90f7342898b949b2045adffa8641c831016 Mon Sep 17 00:00:00 2001
From: SnowCatPDA <snowcat@vpro.ru>
Date: Thu, 26 May 2016 12:33:03 +0500
Subject: [PATCH 1156/1168] UPDATE gitignore

---
 .gitignore | 1 -
 1 file changed, 1 deletion(-)

diff --git a/.gitignore b/.gitignore
index faa1855..b36d023 100644
--- a/.gitignore
+++ b/.gitignore
@@ -3,7 +3,6 @@
 *.bak
 *.orig
 
-.*
 *.o
 *.o.*
 
-- 
2.7.4

